meta-mender-variscite/recipes-bsp/u-boot/patches/imx8mm_var_dart_defconfig.patch

```diff
@@ -0,0 +1,127 @@
+diff -r -u /workdir/var-som-mx8m-mini-dunfell-no-mender/build/tmp/work/imx8mm_var_dart-fslc-linux/u-boot-variscite/1.0-r0/old-src/configs/imx8mm_var_dart_defconfig /workdir/var-som-mx8m-mini-dunfell-no-mender/build/tmp/work/imx8mm_var_dart-fslc-linux/u-boot-variscite/1.0-r0/git/configs/imx8mm_var_dart_defconfig
+--- a/configs/imx8mm_var_dart_defconfig	2023-02-24 04:38:53.145452295 +0000
++++ b/configs/imx8mm_var_dart_defconfig	2023-02-24 04:39:33.986531500 +0000
+@@ -10,8 +10,6 @@
+ CONFIG_SYS_I2C_MXC_I2C2=y
+ CONFIG_SYS_I2C_MXC_I2C3=y
+ CONFIG_SYS_I2C_MXC_I2C4=y
+-CONFIG_ENV_SIZE=0x1000
+-CONFIG_ENV_OFFSET=0x400000
+ CONFIG_DM_GPIO=y
+ CONFIG_TARGET_IMX8MM_VAR_DART=y
+ CONFIG_SPL_MMC_SUPPORT=y
+@@ -59,7 +57,6 @@
+ CONFIG_CMD_FS_GENERIC=y
+ CONFIG_OF_CONTROL=y
+ CONFIG_DEFAULT_DEVICE_TREE="imx8mm-var-dart-customboard"
+-CONFIG_ENV_IS_IN_MMC=y
+ CONFIG_SYS_RELOC_GD_ENV_ADDR=y
+ CONFIG_ENV_VARS_UBOOT_RUNTIME_CONFIG=y
+ CONFIG_NET_RANDOM_ETHADDR=y
+@@ -109,3 +106,32 @@
+ CONFIG_CI_UDC=y
+ CONFIG_SDP_LOADADDR=0x40400000
+ CONFIG_OF_LIBFDT_OVERLAY=y
++CONFIG_ENV_SIZE=0x1000
++CONFIG_ENV_OFFSET=0x400000
++CONFIG_ENV_OFFSET_REDUND=0x800000
++CONFIG_SYS_REDUNDAND_ENVIRONMENT=y
```

Comment:
see above

Reply:
Updated in commit e4d68cbb2d658f327a7ba78b49fb9583c3e7e60f